src/services/content_processor.py

```
@@ -0,0 +1,306 @@
+                        "role": "system",
+                        "content": "You are a helpful assistant that provides concise code example summaries.",
+                    },
+                    {"role": "user", "content": prompt},
+                ],
+                temperature=0.3,
+                max_tokens=100,
+            )
+
+            return response.choices[0].message.content.strip()
+
+        except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)